docs/source/reference/straxen.storage.rst

```diff
@@ -7,7 +7,7 @@ Submodules
 straxen.storage.mongo\_storage module
 -------------------------------------
 
-.. automodule:: straxen.storage.mongo_storage
+.. automodule:: utilix.mongo_storage
```

Comment:
If `mongo_storage.py` is removed, you do not need this section.

Reply:
If possible, please update the docs with https://github.com/XENONnT/straxen/blob/c8b2e2800264ffeae22974b5f9190382d6c65b4f/docs/make_docs.sh

Reply:
That script modifies too many docs that are not relevant to this PR (I think that is worth another PR). So, here I just deleted this section manually.